index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html/css_quiz</title>
+    <link rel="stylesheet" type="text/css" href="quiz.css">
+  </head>
+  <body>
+    <article>
+      <header id="pencil">
+        <h1>Pencil</h1>
+      </header>
+      <div>

```

Comment:
The end of this `<header>` should begin a `<section>`. There should be three sections per article.